{{cookiecutter.repo_name}}/.github/dependabot.yml

```
@@ -0,0 +1,18 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    target-branch: "develop"
```

Poetry updates require a rebase for each update, so we should limit them to one so every accepted PR doesn't require all other Poetry PRs to rebase.

```suggestion
    target-branch: "develop"
    # Limit pull requests to 1 so Dependabot isn't constantly rebasing PRs.
    # https://github.com/python-poetry/poetry/issues/496
    open-pull-requests-limit: 1
```

---

Done